tests/pages/api/register.test.ts

```diff
@@ -267,4 +284,46 @@ describe('register', () => {
             res,
         );
     });
+
+    it('should update user attributes with contactable if given', async () => {
+        authSignInSpy.mockImplementation(() => Promise.resolve(mockAuthResponse));
+        authCompletePasswordSpy.mockImplementation(() => Promise.resolve());
+        authSignOutSpy.mockImplementation(() => Promise.resolve());
+        authUpdateAttributesSpy.mockImplementation(() => Promise.resolve());
+
+        const { req, res } = getMockRequestAndResponse(
+            {},
+            {
+                email: 'test@test.com',
+                password: 'abcdefghi',
+                confirmPassword: 'abcdefghi',
+                nocCode: 'DCCL',
+                regKey: 'abcdefg',
+                checkboxUserResearch: 'checkboxUserResearch',
+            },
+            '',
+            writeHeadMock,
+        );
+
+        await register(req, res);
+
+        const thing = authUpdateAttributesSpy.mock.calls[0][1];
+
+        console.log(thing);
+
+        expect(authUpdateAttributesSpy).toHaveBeenCalledWith('test@test.com', [
+            { Name: 'custom:noc', Value: 'DCCL' },
+            { Name: 'custom:contactable', Value: 'yes' },
+        ]);
```

Comment:
Should this test only care about the above test case? It looks like the below cases are already caught in an earlier test?